apps/bfd-server/bfd-server-launcher/pom.xml

```diff
@@ -119,14 +105,9 @@
 			<!-- Used in tests to verify that requests work as expected. -->
 			<groupId>org.apache.httpcomponents</groupId>
 			<artifactId>httpclient</artifactId>
-			<version>4.5.14</version>
+			<version>${org.apache.httpcomponents.httpclient.version}</version>
 			<scope>test</scope>
-			<exclusions>
-				<exclusion>
-					<artifactId>commons-logging</artifactId>
-					<groupId>commons-logging</groupId>
-				</exclusion>
-			</exclusions>
+			<!--Had to take our exclusion of commons-logging because it was failing -->
```

Comment:
We may have been excluding this for a security reason; let's make sure this is ok.